### PR TITLE
[Reviewer: Adam] Add support for the centos build

### PR DESCRIPTION
**Pull request by eleanor-merry**
Adam, can you please review this change to add support for running monit on centos.

This PR doesn't cover any changes to let the monit scripts run on centos.

### CODE REVIEW

File: debian/clearwater-monit.service [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[Unit]
```
Comment (johadalin) [inline debian/clearwater-monit.service]: Is this file only used in the RPM build, not the deb build?
If so, can we have a comment stating what it's for

File: clearwater.mk [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
+include build-infra/cw-rpm.mk
+
+.PHONY: configure
+configure:
+	./bootstrap && ./configure
 
 .PHONY: deb-build-monit
-deb-build-monit:
+deb-build-monit: configure
 	echo -e "${DEB_COMPONENT} (${DEB_MAJOR_VERSION}-${DEB_MINOR_VERSION}) unstable; urgency=low\n" >debian/changelog
```
Comment (johadalin) [inline clearwater.mk]: I've had a look and see that cw-deb.mk will create the DEB_* variables, but given that we create PKG_* at the top of this file, i'm torn on what we should echo here. 
I don't mind really what we goo for, but it feels slightly simpler/more consistent to change this to PKG_*. If this wont work, then fine.

File: clearwater-monit.root/usr/share/clearwater/clearwater-monit/install/clearwater-monit.postinst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+if [ -f /etc/logrotate.d/monit ]
+then 
+  rm /etc/logrotate.d/monit
+fi
+
+if [ -f /etc/pam.d/monit ]
+then
+  rm /etc/pam.d/monit
+fi
```
Comment (johadalin) [inline clearwater-monit.root/usr/share/clearwater/clearwater-monit/install/clearwater-monit.postinst]: If this addition is the same as what is removed from debian/clearwater-monit.postinst, then all is fine. 
If not, let me know and i'll look at it properly

**Comment by johadalin** [comment]
Couple of comments, but if it builds, installs and uninstalls cleanly, then all fine

File: clearwater-monit.root/usr/share/clearwater/clearwater-monit/install/clearwater-monit.postinst (hunk 3 above)
Comment (eleanor-merry) [inline clearwater-monit.root/usr/share/clearwater/clearwater-monit/install/clearwater-monit.postinst]: Nope, it's the same

**Pull request merged**